server/middleware/searchHandler.js

```diff
@@ -36,7 +36,6 @@ async function handleSearch(req, res, next) {
                 PrevHocsRef: formData['PrevHocsRef'],
                 RefType: formData['RefType'],
                 HomeSecInterest: formData['HomeSecInterest'] === 'true' ? true : undefined,
-                Unworkable: formData['Unworkable'] === 'true' ? true : undefined,
```

Comment:
Is this right to remove this one?

Reply:
This shouldn't be removed as is used for WCS search.